.github/workflows/e2e-tests.yml

```diff
@@ -153,3 +153,47 @@ jobs:
           ./cypress
           ./logs/test.log
         if-no-files-found: ignore
+
+  visual-regression-tests:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    needs: build
+    name: percy-tests-sanity-check-${{ matrix.edition }}
+    env:
+      MB_EDITION: ${{ matrix.edition }}
+      ENTERPRISE_TOKEN: ${{ secrets.ENTERPRISE_TOKEN }}
+    strategy:
+      matrix:
+        java-version: [11]
+        edition: [ee]
+    services:
+      maildev:
+        image: maildev/maildev:1.1.0
```

Comment:
I don't recall if we need/don't need `maildev` for Percy tests.

Reply:
In the main Percy workflow we do so I mimicked it here.

Reply:
We it them for email testing